lib/src/main/java/com/shopify/checkoutsheetkit/CheckoutDialog.kt

```diff
@@ -135,10 +133,7 @@ internal class CheckoutDialog(
 
     private fun hideProgressBar() {
         findViewById<FrameLayout>(R.id.checkoutSdkLoadingSpinner).visibility = GONE
-        findViewById<CheckoutWebViewContainer>(R.id.checkoutSdkContainer).apply {
-            children.firstOrNull()?.scrollY = 0
```

Comment:
Is this no longer needed? It doesn't seem related?

Reply:
moving to a separate PR